esmvaltool/utils/recipe_test_workflow/recipe_test_workflow/site/jasmin.cylc

```diff
@@ -25,12 +29,17 @@
         [[[directives]]]
             --mem = 2G
 
-    [[process<fast>]]
-        execution time limit = PT10M
+    # Resources for recipes that need more than the default.
+    # Variable (fast, medium) must be consistent with flow.cylc.
+    # Comment indicates example recorded usage on Jasmin.
+    [[process<fast=autoassess_landsurface_soilmoisture>]]
+        # Actual: 1m35s, 2.1 GB on 2024-03-29
+        execution time limit = PT3M
```

Comment:
Can this line be removed in favour of the default in `COMPUTE`? 🤔

Reply:
It needs to be more than 2G since example recorded 2.1G.

Reply:
As above, see [1a1da5a](https://github.com/ESMValGroup/ESMValTool/pull/3562/commits/1a1da5a13e0a0e543a824f46f710397b1a1bb3ff)